src/workerd/api/global-scope.h

```diff
@@ -83,10 +83,16 @@ class Navigator: public jsg::Object {
     return 1;
   }
 
+  kj::StringPtr getLanguage() {
+    // Some packages depend on navigator.language being set to a specific value.
+    return "en"_kj;
```

Comment:
just out of curiosity, what about getting the accepted language from the request header (and fallback to "en") as suggested in the issue? would that be possible? or is that something not worth doing?

Reply:
I think that would diverge us from spec and will create unknown side-effects.